pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -101,6 +101,7 @@ class ClassTypeHintMatch:
     "aiohttp_server": "Callable[[], TestServer]",
     "area_registry": "AreaRegistry",
     "async_setup_recorder_instance": "RecorderInstanceGenerator",
+    "calls": "list[ServiceCall]",
```

Comment:
I've seen some PRs that rename fixtures, to prevent confusion with this typing....

But isn't the issue that this name is unclear? Wouldn't `service_calls` not be better?

../Frenck

Reply:
I considered it - but wasn't sure if it was desirable...
Maybe it is.

Reply:
I think, that for most cases, we can make it even a generic/global fixture.

Reply:
There is already a service_calls fixture in template tests - so I've rename it in #118358 
I've added the common fixture in commit d15942561569970a2c12a924d55d7f32bb83e328
I've used the new common fixture in commit f297480871277a00d7b5c0395cf1928f99b00a91